lib/wanda/execution/evaluation.ex

```diff
@@ -82,10 +90,14 @@ defmodule Wanda.Execution.Evaluation do
         message: "Fact gathering ocurred during the execution"
       }
     else
+      computed_values = compute_values(values, facts, env)
```

Comment:
I don't think we need this `compute_values` function. It only has one `Enum` usage. I would directly put here:
```
Enum.map(values, &value_from_conditions(&1, facts, env))
```
And I would name the next function `eval_values`, as we already have a `eval_expectations`

Reply:
+1